### DISCUSSION TITLE
Unable to make QDialog fit container

### DISCUSSION THREAD

**Discussion by Nextgen-developers** [quasarframework/quasar#17331] [General - Components / Directives / etc]
I want the QDialog to fit within its parent container, including the backdrop and other elements. It should not extend to cover the entire window but should remain within the bounds of the parent container. I'm having trouble achieving this behavior. Are there any solutions available?
`<myContainer>
<q-dialog v-model="model">
Dialog content
</q-dialog>
</myContainer>
`

**Comment by ivanjaros** [comment]
Dialog works with window, it is not a child component of any parent but only window, just like notification or tooltip, for example. So this is not possible.
